scripts/postinstall.js

```diff
@@ -0,0 +1,11 @@
+const { spawnSync } = require('child_process');
+const path = require('path');
+
+// git v2.9.0 supports a custom hooks directory. This means we just need to checkin the hooks scripts
+spawnSync('git', ['config', 'core.hooksPath', '.githooks']);
+
+if (!process.env.TF_BUILD && !process.env.UIFABRIC_NO_POSTINSTALL) {
```

Comment:
Is there any way to disable this postinstall step besides setting an environment variable?

Reply:
I'm completely getting rid of it in favor of printing out how to do a builddemo instead of auto running it

Reply:
Nice, I think that's a better option.